### ISSUE TITLE
"exit" keyword not recognized when almond kernel is used in jupyter_console

### ISSUE THREAD

**Issue by mpacer** [almond-sh/almond#240]
When I try to type `exit` in `jupyter_console` while running the `almond` scala kernel, it raises a `scala.NotImplementedError`. 

The error is as follows:

```
In [1]: exit
ERROR InterpreterToIOInterpreter Error when executing exit
scala.NotImplementedError: an implementation is missing
  scala.Predef$.$qmark$qmark$qmark(Predef.scala:284)
  almond.ScalaInterpreter.execute(ScalaInterpreter.scala:457)
  almond.interpreter.InterpreterToIOInterpreter.$anonfun$execute$2(InterpreterToIOInterpreter.scala:66)
  cats.effect.internals.IORunLoop$.cats$effect$internals$IORunLoop$$loop(IORunLoop.scala:85)
  cats.effect.internals.IORunLoop$RestartCallback.apply(IORunLoop.scala:310)
  cats.effect.internals.IORunLoop$RestartCallback.apply(IORunLoop.scala:292)
  cats.effect.IO$$anon$8.run(IO.scala:814)
  java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
  java.lang.Thread.run(Thread.java:748)
ERROR MessageHandler$ Error while processing execute_request message
scala.NotImplementedError: an implementation is missing
  scala.Predef$.$qmark$qmark$qmark(Predef.scala:284)
  almond.ScalaInterpreter.execute(ScalaInterpreter.scala:457)
  almond.interpreter.InterpreterToIOInterpreter.$anonfun$execute$2(InterpreterToIOInterpreter.scala:66)
  cats.effect.internals.IORunLoop$.cats$effect$internals$IORunLoop$$loop(IORunLoop.scala:85)
  cats.effect.internals.IORunLoop$RestartCallback.apply(IORunLoop.scala:310)
  cats.effect.internals.IORunLoop$RestartCallback.apply(IORunLoop.scala:292)
  cats.effect.IO$$anon$8.run(IO.scala:814)
  java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
  java.lang.Thread.run(Thread.java:748)
```

**Comment by mpacer** [comment]
Note <kbd>Ctrl</kbd>-<kbd>D</kbd> twice does work (as you would expect). 

Note this is for when you are running running `jupyter-console` as the way to also launch the kernel (`jupyter console --kernel=scala`), this does not have to do with the issues that arise when connecting to an existing kernel (with `jupyter console --existing`).

**Comment by alexarchambault** [comment]
Fixed by https://github.com/almond-sh/almond/pull/285 (included in the upcoming `0.2.1`).
